new_client/src/components/events/EventList.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+
+const buttonStyle = {
+  margin: '10px 15px',
+  maxWidth: '120px'
+};
+
+const EventList = (props) => {
+  const handleEdit = (eventId) => {
+    props.handleEdit(eventId);
+  };
+
+  const handleDelete = (eventId) => {
+    props.handleDelete(eventId);
+  };

```

Comment:
In addition to deleting an event, can you please show an alert div ? Like __Are you sure you want to delete ?__